proxy/http2/Http2ConnectionState.cc

```diff
@@ -1054,7 +1055,7 @@ Http2ConnectionState::send_a_data_frame(Http2Stream *stream, size_t &payload_len
     return HTTP2_SEND_A_DATA_FRAME_NO_PAYLOAD;
   }
 
-  if (stream->is_body_done() && payload_length < available_size) {
+  if (stream->is_body_done() && payload_length <= available_size) {
```

Comment:
This could be problem when response body size is larger than 16375.
```
$ /opt/nghttp2/bin/nghttp -n "https://127.0.0.1:4443/httpbin/bytes/16376"
Some requests were not processed. total=1, processed=0
```